WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -143,6 +143,21 @@ public class BlockEditorScreen: BaseScreen {
             try FancyAlertComponent().acceptAlert()
         } else {
             publishNowButton.tap()
+            dismissBloggingRemindersAlertIfNeeded()
+        }
+    }
+
+    public func dismissBloggingRemindersAlertIfNeeded() {
+        let bloggingRemindersAlertIsLoaded = app.buttons["Set reminders"].waitForExistence(timeout: 3)
+
+        if bloggingRemindersAlertIsLoaded {
+            if XCUIDevice.isPad {
+                app.swipeDown(velocity: .fast)
+            } else {
+                let dismissBloggingRemindersAlertButton = app.buttons.element(boundBy: 0)
+
+                dismissBloggingRemindersAlertButton.tap()
+            }
```

Comment:
Swift structure nitpick.

To help with readability, it's good to avoid the [_Pyramid of doom_](https://en.wikipedia.org/wiki/Pyramid_of_doom_(programming)). Swift has a neat way to do this by introducing compiler enforced early returns with the [`guard` statement](https://docs.swift.org/swift-book/ReferenceManual/Statements.html#ID524). The cool thing about using `guard` for early returns is that the compiler will fail the build if we forget to `return`, which is super handy.

```suggestion
        guard app.buttons["Set reminders"].waitForExistence(timeout: 3) else { return }

        if XCUIDevice.isPad {
            app.swipeDown(velocity: .fast)
        } else {
            let dismissBloggingRemindersAlertButton = app.buttons.element(boundBy: 0)
            dismissBloggingRemindersAlertButton.tap()
        }
```

---

P.S. Here's a clear example of Pyramid of doom 😄 :

![pyramid of doom image](https://miro.medium.com/max/962/1*YoTPCR_l1ApgGGfMp6ZzmQ.png)

Reply:
Oh yeah, totally forgot about the `guard` statement, I have used it in my trial. Thanks! =D